logging-modules/log4j/src/main/java/com/baeldung/log4j/Log4jComparisonSysout.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.log4j;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintStream;
+
+public class Log4jComparisonSysout {
+
+	public static void main(String[] args) throws FileNotFoundException {
```

Comment:
please change indentation to use spaces, not tabs

Reply:
Updated. Changed to spaces.